src/data/sensor.ts

```diff
@@ -1,2 +1,15 @@
+import { HomeAssistant } from "../types";
+
 export const SENSOR_DEVICE_CLASS_BATTERY = "battery";
 export const SENSOR_DEVICE_CLASS_TIMESTAMP = "timestamp";
+
+export type SensorDeviceClassUnits = { units: string[] };
```

Comment:
Something like
```suggestion
export type SensorDeviceClassUnits = { units: string[]; convertible: boolean; };
```

Reply:
That's what I suggested in the core PR

Reply:
Is there a plan to make use of this list for things other than unit conversion?
If it's only for unit conversion, then I think it's better to simply return an empty list when it's not convertible, as adjusted in https://github.com/home-assistant/core/pull/85213/commits/96321584a208b6d79d638eb01565001d09043662.

Reply:
I updated the endpoint et removed the hardcoded list in frontend